concent_api/core/queue_operations.py

```diff
@@ -1,2 +1,19 @@
-def send_blender_verification_request():  # TODO implement sending real verivication_request to Work Queue
-    pass
+from conductor.tasks import blender_verification_request
+from utils.helpers import get_storage_result_file_path
+from utils.helpers import get_storage_source_file_path
+
+
+def send_blender_verification_request(compute_task_def):
+    task_id = compute_task_def['task_id']
+    subtask_id = compute_task_def['subtask_id']
+    source_package_path = get_storage_source_file_path(subtask_id, task_id)
+    result_package_path = get_storage_result_file_path(subtask_id, task_id)
+    output_format = compute_task_def['extra_data']['output_format']
+    scene_file = compute_task_def['extra_data']['scene_file']
+    blender_verification_request(
```

Comment:
sholdn't `apply` or `delay` be used here?

Reply:
`delay`, `apply` is for synchronous call.

Reply:
ok, from what I read (http://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.delay), `apply_async` and `delay` are for asynchronous calls, `apply` is for synchronous. Here we don't call any of them. What is called by default?

Reply:
Ok, I just read (http://docs.celeryproject.org/en/latest/userguide/calling.html#basics) that _calling_ the task will execute it locally (in the current process), withou sending the message to the queue. Is this what we want here?

Reply:
I think that we want `delay` here.

Reply:
I've added it before you post this, just to see if it works. You can check if everything is implemented correctly ;)